lib/features/mapping/domain/entities/Point.dart

```diff
@@ -16,6 +16,12 @@ class Point extends ValueObject<Map<String, dynamic>> {
           PointType.point,
         ]);
 
+  @override
```

Comment:
I think these are redundant, since ValueObject extends JsonObjects which implements Equatable already. There are probably more of these around.

Reply:
Have removed this and similar redundant examples from 27 different files. Should be corrected now.